crates/red_knot_test/src/lib.rs

```diff
@@ -118,7 +118,7 @@ fn run_test(
             }
 
             assert!(
-                matches!(embedded.lang, "py" | "pyi" | "text"),
+                matches!(embedded.lang, "py" | "pyi" | "python" | "text"),
                 "Supported file types are: py, pyi, text"
```

Comment:
I think the assertion message may now need to be updated to mention `python`? Which means we may also want to change the message so it talks about "language tags" rather than "file types"